src/test/java/com/wikia/webdriver/common/core/helpers/Emulator.java

```diff
@@ -15,6 +15,15 @@
           .put("mobile", false)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "mobile" 8 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)